### ISSUE TITLE
Support for Shotwell

### ISSUE THREAD

**Issue by GoogleCodeExporter** [yendo/gphotoframe#61]
```
Shotwell - http://yorba.org/shotwell/
```

Original issue reported on code.google.com by `yendo0206` on 13 Sep 2010 at 7:13

**Comment by GoogleCodeExporter** [comment]
Original comment by `yendo0206` on 17 Sep 2010 at 1:57
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in revision a225a99010.
```

Original comment by `yendo0206` on 18 Sep 2010 at 4:49
- Changed state: **Fixed**
